account/account.go

```diff
@@ -44,60 +48,85 @@ type Config struct {
 	PrivateKey string
 	// KeystorePath path of keystore
 	KeystorePath string
-	// Password keystore password
-	Password string
+	// PasswordFilePath stores keystore password
+	PasswordFilePath string
 	// Address public address of sentry wallet
 	Address string
 }
 
+type baseAccount struct {
+	balance *big.Int
+	nonce   uint64
+	address common.Address
+}
+
+func (a *baseAccount) Address() common.Address {
+	return a.address
+}
+
+func (a *baseAccount) SetBalance(balance *big.Int) {
+	a.balance = balance
+}
+
+func (a *baseAccount) GetBalance() *big.Int {
+	return a.balance
+}
+
+func (a *baseAccount) SetNonce(nonce uint64) {
+	a.nonce = nonce
+}
+
+func (a *baseAccount) GetNonce() uint64 {
+	return a.nonce
+}
+
 type keystoreAccount struct {
 	keystore *keystore.KeyStore
 	account  accounts.Account
+	*baseAccount
 }
 
-func newKeystoreAccount(keystorePath, password, opAccount string) (*keystoreAccount, error) {
+func newKeystoreAccount(keystorePath, passwordFilePath, opAccount string) (*keystoreAccount, error) {
+	address := common.HexToAddress(opAccount)
 	ks := keystore.NewKeyStore(keystorePath, keystore.StandardScryptN, keystore.StandardScryptP)
-	account, err := ks.Find(accounts.Account{Address: common.HexToAddress(opAccount)})
+	account, err := ks.Find(accounts.Account{Address: address})
 	if err != nil {
-		log.Errorw("failed to create key store account", "err", err)
+		log.Errorw("failed to create key store baseAccount", "err", err)
 		return nil, err
 	}
 
-	err = ks.Unlock(account, password)
-	if err != nil {
-		log.Errorw("failed to unlock account", "err", err)
-		return nil, err
-	}
+	passwords := utils.MakePasswordListFromPath(passwordFilePath)
```

Comment:
why passwords is a slice? I thought it is one item

Reply:
all validators's passwords could be in one file

Reply:
fixed